### DISCUSSION TITLE
Realtime not working as on 19th April 2024

### DISCUSSION THREAD

**Discussion by kaushalkumar86** [supabase/supabase#22903] [Questions]
Realtime is not working, the same code was was working for a year up until yesterday, today its not working, I mailed and asked supabase, that said no issues from their end.
Anything changed since yesterday?
Realtime channels are created but no postgrest changes are received. Even broadcast is also not working, no msgs are received on broadcast.

**Comment by GaryAustin1** [comment]
My test realtime using postgres changes is working.
Any errors from .subscribe()?
Any errors in the realtime logs?

**Comment by kaushalkumar86** [comment]
It's working now.
Got update from supabase team
